examples/grpc/common/src/main/java/io/helidon/grpc/examples/common/GreetClientApi.java

```diff
@@ -0,0 +1,81 @@
+ * limitations under the License.
+ */
+
+package io.helidon.grpc.examples.common;
+
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+import io.helidon.grpc.client.ClientRequestAttribute;
+import io.helidon.grpc.client.ClientServiceDescriptor;
+import io.helidon.grpc.client.ClientTracingInterceptor;
+import io.helidon.grpc.client.GrpcServiceClient;
+import io.helidon.grpc.examples.common.Greet.GreetRequest;
+import io.helidon.grpc.examples.common.Greet.GreetResponse;
+import io.helidon.grpc.examples.common.Greet.SetGreetingRequest;
+import io.helidon.grpc.examples.common.Greet.SetGreetingResponse;
+import io.helidon.tracing.TracerBuilder;
+import io.opentracing.Tracer;
+
+import java.net.URI;
+
+/**
+ * A client for the {@link GreetService} implemented with Helidon gRPC client API.
+ *
+ * @author Bin Chen
+ */
+public class GreetClientApi
+    {
+
+    private GreetClientApi() { }
+
+    /**
+     *  The program entry point.so
+     * @param args  the program arguments
+     *
+     * @throws Exception  if an error occurs
+     */
+    public static void main(String[] args) throws Exception {
+        Tracer tracer = TracerBuilder.create("Client")
+                .collectorUri(URI.create("http://localhost:9411/api/v2/spans"))
+                .build();
+
+        ClientTracingInterceptor tracingInterceptor = ClientTracingInterceptor.builder(tracer)
+                .withVerbosity().withTracedAttributes(ClientRequestAttribute.ALL_CALL_OPTIONS).build();
+
+        ClientServiceDescriptor descriptor = ClientServiceDescriptor
+                .builder(GreetServiceGrpc.getServiceDescriptor())
+                .intercept(tracingInterceptor)
+                .build();
+
+        Channel channel = ManagedChannelBuilder.forAddress("localhost", 1408)
+                .usePlaintext()
+                .build();
+
+        GrpcServiceClient grpcClient = GrpcServiceClient.create(channel, descriptor);
+
+        // blocking unary
+        System.out.println(grpcClient.<GreetRequest, GreetResponse>blockingUnary("Greet", GreetRequest.newBuilder().setName("Aleks").build()));
+        System.out.println(grpcClient.<SetGreetingRequest, SetGreetingResponse>blockingUnary("SetGreeting", SetGreetingRequest.newBuilder().setGreeting("Ciao").build()));
+        System.out.println(grpcClient.<GreetRequest, GreetResponse>blockingUnary("Greet", GreetRequest.newBuilder().setName("Aleks").build()));
+
+        // asynchronous unary
+        System.out.println(grpcClient.<GreetRequest, GreetResponse>unary("Greet", GreetRequest.newBuilder().setName("Aleks").build()).get());
+        System.out.println(grpcClient.<SetGreetingRequest, SetGreetingResponse>unary("SetGreeting", SetGreetingRequest.newBuilder().setGreeting("Ciao").build()).get());
+        System.out.println(grpcClient.<GreetRequest, GreetResponse>unary("Greet", GreetRequest.newBuilder().setName("Aleks").build()).get());
+
+        Thread.sleep(5000);
```

Comment:
Do we need this sleep here, what is it for? If it is here for a reason then add a comment above it saying why this example needs a sleep. If the example works without it then remove it.